util/named_requirement_verification/legacy_bidirectional_iterator.h

```diff
@@ -0,0 +1,145 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides class to verify conformity with named requirement
+//  LegacyBidirectionalIterator
+//
+*******************************************************************************/
+
+#ifndef __SYCLCTS_TESTS_ITERATOR_REQUIREMENTS_LEGACY_BIDIRECTIONAL_ITERATOR_H
+#define __SYCLCTS_TESTS_ITERATOR_REQUIREMENTS_LEGACY_BIDIRECTIONAL_ITERATOR_H
+
+#include "common.h"
+#include "legacy_forward_iterator.h"
+
+namespace named_requirement_verification {
+
+/**
+ * @brief Class helps to verify conformity to LegacyBidirectionalIterator named
+ * requirement. Safe to use inside kernel with SYCL 2020.
+ *
+ */
+class legacy_bidirectional_iterator_requirement {
+ public:
+  // Will be used as size of container for error messages
+  // Value should be equal to the number of add_error invocations
+  // Don't forget to update this value if there is any changes in class

```

Comment:
May we add something like this `ITERATOR_REQUIREMENTS_UPDATE_ME` to provide easier way to find all places that should be updated in such case

Reply:
It's actually a good idea, but since there is only one place in every requirement verifications that have to be updated in case of changes, we can leave it as it is with comment section.